README.md

```diff
@@ -13,7 +13,7 @@ This project is licensed under the Apache-2.0 License.
 
 ## Building
 
-Kotlin Multiplatform projects are in [Alpha](https://kotlinlang.org/docs/reference/evolution/components-stability.html). The CRT interfaces are subject to change.
+CRT interfaces are subject to change.
 
```

Comment:
**Suggestion**: We should add a note to the readme about needing to fetch/update submodules.

Reply:
Yeah this is all a WIP right now as I figure out the build instructions for local vs CI.